src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -166,22 +166,8 @@ public PagedResponse<Admission> getAdmissionsPageable(LocalDateTime dateFrom, Lo
 	 * @return the list of Admissions (could be empty)
 	 * @throws OHServiceException
 	 */
-	public List<Admission> getAdmissionsByDate(LocalDateTime dateFrom, LocalDateTime dateTo) throws OHServiceException {
-		return ioOperations.getAdmissionsByAdmDate(dateFrom, dateTo);
-	}
-
-	/**
-	 * Method that returns the list of completed Admissions (Discharges) not logically deleted
-	 * within the specified date range, divided by pages
-	 * @param dateFrom
-	 * @param dateTo
-	 * @param page
-	 * @param size
-	 * @return the list of completed Admissions (could be empty)
-	 * @throws OHServiceException
-	 */
-	public List<Admission> getDischarges(LocalDateTime dateFrom, LocalDateTime dateTo, int page, int size) throws OHServiceException {
-		return ioOperations.getAdmissionsByDischargeDate(dateFrom, dateTo, PageRequest.of(page, size));
+	public List<Admission> getAdmissionsByAdmissionDate(LocalDateTime dateFrom, LocalDateTime dateTo) throws OHServiceException {
+		return ioOperations.getAdmissionsByAdmissionDate(dateFrom, dateTo);
```

Comment:
Not sure here... is `getDischarges` disappeared now?

Reply:
yes

Reply:
ok, right, we have the `getDischargesPageable` now, thanks!